### PR TITLE
Add gitaly pv pvc creation

### PR DESCRIPTION
**Pull request by kcatro**

### CODE REVIEW

File: gitlab.tf [hunk 1]
```diff
@@ -92,6 +92,44 @@ data "template_file" "helm_values" {
   }
 }
 
+resource "kubernetes_persistent_volume" "gitaly_pv" {
```
Comment (filipowm) [inline gitlab.tf]: Is this responsible for creating a disk for gitaly? Or is it created somewhere else?

File: gke.tf [hunk 2]
```diff
@@ -103,7 +103,7 @@ resource "kubernetes_storage_class" "pd-ssd" {
   }
 
   storage_provisioner = "kubernetes.io/gce-pd"
-
+  reclaim_policy      = "Retain"
```
Comment (filipowm) [inline gke.tf]: Potential conflict with other PR

File: gitlab.tf (hunk 1 above)
Comment (kcatro) [reply gitlab.tf]: It is assumed the disk is created outside the scope of this module (so e.g. it's a leftover from a previous installation which got deleted). I've added a README portion about this.

File: gke.tf (hunk 2 above)
Comment (kcatro) [reply gke.tf]: Rebased.

**Pull request merged**